vlm/src/test/scala/geotrellis/contrib/vlm/avro/GeotrellisRasterSourceSpec.scala

```diff
@@ -34,8 +34,8 @@ class GeotrellisRasterSourceSpec extends FunSpec with RasterMatchers with Better
   val uriMultiband = s"file://${TestCatalog.multibandOutputPath}"
   val uriSingleband = s"file://${TestCatalog.singlebandOutputPath}"
   val layerId = LayerId("landsat", 0)
-  val sourceMultiband = GeotrellisRasterSource(uriMultiband, layerId)
-  val sourceSingleband = GeotrellisRasterSource(uriSingleband, layerId)
+  lazy val sourceMultiband = GeotrellisRasterSource(uriMultiband, layerId)
```

Comment:
A pretty important thing here, it will be evaluated before `beforeAll` function call.

Reply:
Ah, I wasn't aware of that :+1:

Reply:
@jbouffard actually it's due to regression that was fixed here: https://github.com/geotrellis/geotrellis-contrib/pull/115/commits/d1cffd2393c350cb46de1eb28de8742ac5a9d863